scripts/composer/post-install.sh

```diff
@@ -1,21 +1,32 @@
 #!/bin/sh
 
+DOCUMENTROOT=web
```

Comment:
One way to do this is by allowing the user to save a settings file that contains the configuration and include this if it exists, something like this:

```
# Load config file if it exists.
if [ -f config.sh ]; then
  . config.sh
fi

if [ -z "$DOCUMENTROOT" ]; then DOCUMENTROOT=web; fi
```

But this might be too much, you can also simply provide a symlink if the the docroot needs to be somewhere else.